extensions/wikia/AdEngine/AdEngine2Hooks.class.php

```diff
@@ -112,7 +111,7 @@ public static function onInstantGlobalsGetVariables( array &$vars ) {
 		$vars[] = 'wgAdDriverTurtleCountries';
 		$vars[] = 'wgAdDriverWadBTCountries';
 		$vars[] = 'wgAdDriverWadILCountries';
-		$vars[] = 'wgAdDriverPreFooterAndBLBSwitchedCountries';
+		$vars[] = 'wgAdDriverStickyAndSwapCountries';
```

Comment:
Yeah, this place is a pain. You need to rename it to `wgAdDriverMobileStickyAndSwapCountries` and add one more line with `wgAdDriverBfabStickinessOasisCountries`, otherwise they're not instant globals :(